copy/copy.go

```diff
@@ -178,6 +182,9 @@ func Image(ctx context.Context, policyContext *signature.PolicyContext, destRef,
 		// For now, use DestinationCtx (because blob reuse changes the behavior of the destination side more); eventually
 		// we might want to add a separate CommonCtx — or would that be too confusing?
 		blobInfoCache: blobinfocache.DefaultCache(options.DestinationCtx),
+
+		compressionFormat: *options.DestinationCtx.CompressionFormat,
```

Comment:
The dereference breaks consumers of c/image. We need to change that to avoid nil dereference.